rally_test.go

```diff
@@ -16,6 +16,20 @@ import (
 
 const token = "token"
 
+func TestGetError(t *testing.T) {
```

Comment:
Could this method be given a more explanatory name; e.g. ``TestFailReadResponseBody``? Also it should be moved further below in the file, after ``TestFailStatusCodeNotOK``. The first tests in the file is the sunshine scenarios.